scripts/build/TestImpactAnalysis/tiaf_driver.py

```diff
@@ -35,7 +35,8 @@ def test_failure_policy(value):
 
     parser = argparse.ArgumentParser()
     parser.add_argument('--config', dest="config", type=file_path, help="Path to the test impact analysis framework configuration file", required=True)
-    parser.add_argument('--destBranch', dest="dst_branch",  help="For PR builds, the destination branch to be merged to, otherwise empty")
+    parser.add_argument('--srcBranch', dest="src_branch",  help="The branch that is being build", required=True)
+    parser.add_argument('--dstBranch', dest="dst_branch",  help="For PR builds, the destination branch to be merged to, otherwise empty")
     parser.add_argument('--branchesOfTruth', dest="branches_of_truth", type=lambda arg: arg.split(','), help="Comma separated branches that seeding will occur on", required=True)
     parser.add_argument('--pipeline', dest="pipeline", help="Pipeline the test impact analysis framework is running on", required=True)
     parser.add_argument('--pipelinesOfTruth', dest="pipelines_of_truth", type=lambda arg: arg.split(','), help="Comma separated pipeline that seeding will occur on", required=True)
```

Comment:
Python arguments normally don't normally use snakeCase but dashes between arguments
i.e `dest-branch` instead of `destBranch` or `branches-of-truth` instead of `branchesOfTruth`

Reply:
Agreed, I will change to the more Pythonic `separate-with-dashes` convention 👍